chat_filter.user.js

```diff
@@ -814,32 +814,17 @@ add_initializer(function(){
 
     add_custom_css([
         ".chat-room { z-index: inherit !important; }",
+
         ".chat-settings { z-index: 100 !important; }",
+        ".chat-settings { max-height: 500px; }",
```

Comment:
Yup. Just saying that since I saw that `.custom_list_menu li` did have everything in one line